dvc/command/data_sync.py

```diff
@@ -97,7 +97,8 @@ def shared_parent_parser():
         "-j",
         "--jobs",
         type=int,
-        help="Number of jobs to run simultaneously.",
+        help="Number of jobs to run simultaneously. "
+        "Refer to docs to get default value.",
```

Comment:
Maybe let's just copypaste from docs? E.g.
```
The default value is 4 * cpu_count(). For SSH remotes, the default is 4. Using more jobs may improve the overall transfer speed.
```

Reply:
I've noticed that most help messages are short (2 sentences max) so I decided to use "refer to docs...". However, I have no strong opinion in that regard. If it's fine to have long messages I can copy text from cmd ref.

Reply:
Yes, there is no limit for the help messages. We could totally copypaste those two sentences :) Thank you! :pray:

Reply:
I've updated message